app/scripts/modules/core/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@spinnaker/core",
+  "version": "0.0.1",
+  "main": "lib/lib.js",
+  "typings": "lib/src/index.d.ts",
```

Comment:
typically the main entry and typings entry should be in the same directory.  Why is this different?